charts/newrelic-infrastructure/values.yaml

```diff
@@ -68,8 +68,32 @@ serviceAccount:
   # Specify any annotations to add to the ServiceAccount
   annotations:
 
-# If you wish to provide additional labels to apply to the pod(s), specify
-# them here
+# For unprivileged mode.  Default is 1000.
+# Can also be modified for OpenShift clusters.
+# https://www.openshift.com/blog/a-guide-to-openshift-and-uids
+runAsUser:
+
+### Openshift 4.x ###
+# Set openshift: true if installing in an OpenShift 4.x Cluster
+openshift: false
+
+# Openshift 4.x control plane endpoints
```

Comment:
I would add some documentation explaining the different usage of this endpoints, can we configure them automatically?

Reply:
I could add in some logic that would configure these automatically if `openshift: true`.  We'd still need to allow them to be overridden though in case non-standard endpoints are being used.  I'll rework this.